frontend/src/components/dropdown.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 
 import { useOnClickOutside } from '../hooks/UseOnClickOutside';
```

Comment:
Please consider removing the useOnClickOutside declaration as it is currently unused. This is causing the CI/CD build to fail.

```suggestion

```

Reply:
Sure!